agent/util-scripts/pbench-move-results

```diff
@@ -8,8 +8,24 @@ pbench_bin="$(realpath -e ${script_path}/..)"
 . "${pbench_bin}"/base
 
 function usage() {
-    printf "usage:\n"
-    printf "${script_name} [--help] [--controller=<controller>] [--user=<user>] [--prefix=<path>] [--xz-single-threaded] [--show-server]\n"
+    printf -- "usage:\n%s [--help] --user=<user> [--controller=<controller>]" "${script_name}"
+    printf -- " [--prefix=<path>] [--xz-single-threaded] [--show-server]\n"
+    printf -- "\nThe '--user' option value is required, and can also be provided\n"
+    printf -- "by the 'PBENCH_USER' environment variable.\n"
```

Comment:
"The '--user' option value is required if not provided by the 'PBENCH_USER' environment variable; otherwise, the value provided on the command line will override any value provided by the environment."